### PR TITLE
History front end

### LINKED ISSUE #41: History
front end history

 - [ ] Route
Back-end route `/api/employees`
method: `GET`
---
 - [ ] Description
    -display all employees that have been reviewed before
    -  buttons to view form for specific a employee

### PR DESCRIPTION
**Pull request by rubasider**
Call the history with http://localhost:3000/dashboard/history endpoint.

on this page you can see a list of employees that have been reviewed before

Does this Fix an Issue? Fixes #41

How to test this PR npm i

cd client && npm i

cd..

npm run dev

go to http://localhost:3000/dashboard/history endpoint

Screenshots of changes if the database is empty you will only see a page loading
https://ibb.co/1v0qyT6

### CODE REVIEW

File: client/src/components/LayoutComponents/Dashboard/HistoryStyle.jsx [hunk 1]
```diff
@@ -10,10 +10,27 @@ export default makeStyles((theme) => ({
 		display: 'flex',
 		flexWrap: 'wrap',
 	},
-	textField: {
+	// textField: {
```
Comment (yaqootturman) [inline client/src/components/LayoutComponents/Dashboard/HistoryStyle.jsx]: delete comments

File: client/src/components/LayoutComponents/Dashboard/NewMemberStyle.jsx [hunk 2]
```diff
@@ -1,18 +1,39 @@
 import { makeStyles } from '@material-ui/core/styles';
 export default makeStyles((theme) => ({
-	root: {
+	form: {
```
Comment (yaqootturman) [inline client/src/components/LayoutComponents/Dashboard/NewMemberStyle.jsx]: Make it responsive!

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by yaqootturman — DISMISSED** [review]

File: client/src/components/LayoutComponents/Dashboard/History.jsx [hunk 3]
```diff
@@ -48,22 +49,28 @@ export default function AddNewMember() {
...
-							<Link to={`/dashboard/history/${empValue.id}`}>view</Link>"
-						</Button>
-					</ListItem>
-
-					<Divider variant="fullWidth" component="li" />
+					<Box boxShadow={2}>
+						<ListItem className={classes.listItem}>
+							<ListItemAvatar>
+								<Avatar alt="Remy Sharp" src="https://ibb.co/1fJgHrv" />
+								{/* <Person color="disabled" fontSize="large" /> */}
```
Comment (mantagen) [inline client/src/components/LayoutComponents/Dashboard/History.jsx]: :fire:

File: client/src/components/LayoutComponents/Dashboard/History.jsx [hunk 4]
```diff
@@ -48,22 +49,28 @@ export default function AddNewMember() {
...
+								onClick={handleClick}
+								color="text.secondary"
+								variant="contained"
+								size="larg"
+							>
+								<Link to={`/dashboard/history/${empValue.id}`}> المقيمين</Link>
+							</Button>
+						</ListItem>
+					</Box>
+					{/* <Divider variant="fullWidth" component="li"
```
Comment (mantagen) [inline client/src/components/LayoutComponents/Dashboard/History.jsx]: :fire_extinguisher:

File: client/src/components/LayoutComponents/Dashboard/NewMember.jsx [hunk 5]
```diff
@@ -53,113 +72,177 @@ export default function AddNewMember() {
...
-					input.message = '';
-					input.isValid = true;
-				}
-			}
+		if (empData.email && !validateEmail(empData.email.value)) {
+			empData.email.message = 'Enter a valid email please';
+			swal('error', empData.email.message, 'error');
+			empData.email.isValid = false;
+		} else {
+			const TheData = {
```
Comment (mantagen) [inline client/src/components/LayoutComponents/Dashboard/NewMember.jsx]: `theData` (or just `data`) is fine

File: client/src/components/LayoutComponents/Dashboard/NewMember.jsx [hunk 6]
```diff
@@ -53,113 +72,177 @@ export default function AddNewMember() {
...
+										}}
+									/>
+								</Box>
+								<Box m={2}>
+									<FormControl className={classes.fullWidth}>
+										<InputLabel
+											htmlFor="position"
+											color="primary"
+											placeholder="	المهنة"
+											// position="end"
```
Comment (mantagen) [inline client/src/components/LayoutComponents/Dashboard/NewMember.jsx]: :eagle:

File: client/src/components/LayoutComponents/Dashboard/HistoryStyle.jsx [hunk 7]
```diff
@@ -10,10 +10,22 @@ export default makeStyles((theme) => ({
...
-	textField: {
+
+	listItem: {
+		bgcolor: 'info.main',
 		display: 'flex',
-		marginLeft: theme.spacing(1),
-		marginRight: theme.spacing(1),
-		width: 200,
+		width: '94%',
+		display: 'flex',
```
Comment (mantagen) [inline client/src/components/LayoutComponents/Dashboard/HistoryStyle.jsx]: Same as line 16

**Review by mantagen — CHANGES_REQUESTED** [review]

**Review by yaqootturman — DISMISSED** [review]

**Review by karmelyoei — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**